Formula/vgrep.rb

```diff
@@ -1,8 +1,8 @@
 class Vgrep < Formula
   desc "User-friendly pager for grep"
   homepage "https://github.com/vrothberg/vgrep"
-  url "https://github.com/vrothberg/vgrep/archive/v2.3.3.tar.gz"
-  sha256 "062145687d4c33f66b35be15633ff60cd24fd467bf2791f1a8c3ffb069935aa4"
+  url "https://github.com/vrothberg/vgrep/archive/v2.4.0.tar.gz"
+  sha256 "d0dd594156638158f7163d37169dd02e3a82eabfa6c8f57d6eb1dfa89c669c03"
   license "GPL-3.0"
```

Comment:
```suggestion
  license "GPL-3.0-only"
```
(guessed from https://github.com/vrothberg/vgrep/blob/6f62e04163b91b4ff5c099d54340adb920bfb4d6/vgrep.go#L5)